src/PrestaShopBundle/Bridge/Helper/Listing/Field/Field.php

```diff
@@ -56,6 +56,11 @@ class Field implements FieldInterface
      * - title: which is required and represents the text, will be shown in the list's header
      * - width: define the width for the cell of this field
      *
+     * @todo: class unused. Options resolver should be moved.
+     *      Where could we put fieldsList options resolver? to configurator?
+     *      (if it would have a setter we could put it straight in configuration, but it doesn't because its used by reference in hook)
+     *      check HelperListConfiguration->fieldsList usage
```

Comment:
should do smth similar as with actions. Just actions was made private and added public methods like "addRowAction" in which we could use options resolver. In this case im not sure.. Probably we could move the resolver to ListConfigurator (but then should we move for all the actions too?)

Reply:
fieldsList is public prop (cuz its provided in some hooks with reference), so there is probably no point having a method with "protection of optionsResolver" when the field is public and can be changed directly anyway